docs/user/pixel_observations.md

```diff
@@ -0,0 +1,122 @@
+## Using Image Observations
+Although most environments return observations that somehow describe the physical state of the agent and environment, others return observations that contain spatial data instead. As an example, consider the `MemorizeDigits-v0` gym environment, where the agent must memorize what each digit looks like using RGB observations such as:
+
+```
+  ---------------------------
+ |                           |
+ |         ******            |
+ |         ******            |
+ |       **      **          |
+ |       **      **          |
+ |               **          |
+ |               **          |
+ |           ****            |
+ |           ****            |
+ |       ****                |
+ |       ****                |
+ |       **********          |
+ |       **********          |
+ |                           |
+  ---------------------------
+```
+
+Image observations can be represented as tensors containing each individual pixel's RGB value. Typically these tensors have the shape `(HEIGHT, WIDTH, CHANNELS)` where CHANNELS denotes the number of channels. RGB images have 3 channels and grayscale images have 1 channel.
+
+This page will guide you through managing these image observations and environments.
+## Pixel Environments
+The following gym environments have been tested with garage and have examples written for them:
+ - `MemorizeDigits-v0` - see  `examples/tf/ppo_memorize_digits.py`
+ - `CubeCrash-v0`- see `examples/tf/trpo_cubecrash.py`
+ - `PongNoFrameskip-v4` - see `examples/tf/dqn_pong.py`
+
+ Gym environments that do not return pixel observations by default can be used with `garage.envs.wrappers.PixelObservationWrapper`, which overrides the default observation and instead returns a tensor of RGB values by making calls to `env.render(mode="rgb_array")`. See the section on this wrapper below for more on this.
+
+## Environment Wrappers
+
+Garage includes various environment wrappers that are particularly useful when working with pixel observations. You may use multiple wrappers simultaneously, though the order in which you use them matters. See the pixel-specific wrappers section for more details on this.
+
+### GarageEnv
+When wrapping image-based environments in `GarageEnv`, `is_image` should be set to `True`:
+```python
+env = GarageEnv(gym.make("MemorizeDigits-v0"), is_image=True)
```

Comment:
we always use single quotes in garage code, so let's keep that consistent here

Reply:
Fixed